### ISSUE TITLE
Crashes on startup

### ISSUE THREAD

**Issue by Wandelstock** [Kunzisoft/KeePassDX#722]
**Describe the bug**
When opening {(or starting) the database (or app), keepass crashes

** Keepass Database **
 - Created with: Original Keepass
 - Version: long time ago
 - Location: gdrive

**KeePassDX (please complete the following information):**
 - Version: latest pro version 2.8.6

**Android (please complete the following information):**
 - Device: motorola g6plus, but also on other devices
 - Version: 9, Pie, also on other android versions

**Additional context**
Send bug report

**Comment by hkzlab** [comment]
Also happening on a Motorola One Zoom, Android 10, database synced via Nextcloud.
Opens fine with Nextcloud keepass plugin.

**Comment by Stephan-P** [comment]
On my Samsung A41 (Android 10) the app doesn't crash, but just fails to open the database after I enter the correct password. Returns to opening screen.

**Comment by J-Jamet** [comment]
Seems to be a big bug in the new google API which makes the code work only on some devices after upgrading to API 30.

I downgrade to API 29 before making a new release.

```
2020-09-26 10:10:58.487 4754-4754/com.kunzisoft.keepass.pro E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.kunzisoft.keepass.pro, PID: 4754
    java.lang.NoSuchMethodError: No virtual method getTextSizeUnit()I in class Landroid/widget/TextView; or its super classes (declaration of 'android.widget.TextView' appears in /system/framework/framework.jar:classes2.dex)
        at com.kunzisoft.keepass.adapters.NodeAdapter.onBindViewHolder(NodeAdapter.kt:316)
        at com.kunzisoft.keepass.adapters.NodeAdapter.onBindViewHolder(NodeAdapter.kt:52)
        at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:6781)
        at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:6823)
        at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:5752)
        at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6019)
        at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5858)
        at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5854)
        at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2230)
        at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1557)
        at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1517)
        at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:612)
        at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:3924)
        at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:3641)
        at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4194)
        at android.view.View.layout(View.java:17523)
        at android.view.ViewGroup.layout(ViewGroup.java:5612)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:17523)
        at android.view.ViewGroup.layout(ViewGroup.java:5612)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:17523)
        at android.view.ViewGroup.layout(ViewGroup.java:5612)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1741)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1585)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1494)
        at android.view.View.layout(View.java:17523)
        at android.view.ViewGroup.layout(ViewGroup.java:5612)
        at com.google.android.material.appbar.HeaderScrollingViewBehavior.layoutChild(HeaderScrollingViewBehavior.java:142)
        at com.google.android.material.appbar.ViewOffsetBehavior.onLayoutChild(ViewOffsetBehavior.java:41)
        at com.google.android.material.appbar.AppBarLayout$ScrollingViewBehavior.onLayoutChild(AppBarLayout.java:1556)
        at androidx.coordinatorlayout.widget.CoordinatorLayout.onLayout(CoordinatorLayout.java:888)
        at android.view.View.layout(View.java:17523)
        at android.view.ViewGroup.layout(ViewGroup.java:5612)
        at androidx.constraintlayout.widget.ConstraintLayout.onLayout(ConstraintLayout.java:1915)
        at android.view.View.layout(View.java:17523)
        at android.view.ViewGroup.layout(ViewGroup.java:5612)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:17523)
        at android.view.ViewGroup.layout(ViewGroup.java:5612)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1741)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1585)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1494)
        at android.view.View.layout(View.java:17523)
        at android.view.ViewGroup.layout(ViewGroup.java:5612)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:17523)
```

**Comment by J-Jamet** [comment]
Thanks for your quick feedback.
 I find it maddening to see that the upgrade of a new API removes methods without prevention. I tested well on my devices freshly updated to Android 11 before making the 2.8.6 release and I had no problem. But I was able to reproduce the crash on an old emulator (I missed it).

Version 2.8.7 should be available after validation by Google shortly on the Play Store.

Thank you again for your responsiveness, it is thanks to you that I can correct the problems quickly.
